guides/v2.2/cloud/project/new-relic.md

```diff
@@ -83,52 +98,56 @@ To add a license key to an environment:
 
 1. Set the variable.
 
-    ```bash
-    magento-cloud variable:set php:newrelic.license <newrelic-license-key>
-    ```
+   ```bash
+   magento-cloud variable:set php:newrelic.license <newrelic-license-key>
+   ```
 
-    {:.bs-callout .bs-callout-info}
-    Adding a variable to an environment triggers a redeployment to update the environment variables and values. See [Working with variables]({{page.baseurl}}/cloud/env/working-with-variables.html).
+   {:.bs-callout-info}
+   Adding a variable to an environment triggers a redeployment to update the environment variables and values. See [Working with variables]({{page.baseurl}}/cloud/env/working-with-variables.html).
 
-1. Log in to your New Relic account and verify it is receiving data.
+1. [Log in to New Relic](https://login.newrelic.com/login) to verify that it is receiving data.
 
 {:.procedure}
 To remove a license key from an environment:
 
 1. List environment variables.
 
-    ```bash
-    magento-cloud vget
-    ```
+   ```bash
+   magento-cloud vget
+   ```
 
-    Response:
+   Response:
 
-    ```terminal
-    +----------------------+-------------+------------------------+
-    | Name                 | Level       | Value                  |
-    +----------------------+-------------+------------------------+
-    | php:newrelic.license | environment | <newrelic-license-key> |
-    +----------------------+-------------+------------------------+
-    ```
-    {: .no-copy}
+   ```terminal
+   +----------------------+------------   +------------------------+
```

Comment:
Seems like some spaces were introduced in this sample response that makes it look bad.